conngen/conngenmodule.cpp

```diff
@@ -365,10 +353,10 @@ ConnectionGeneratorModule::CGStart_cgFunction::execute(
    purposes. Usually, connections are created from a
    ConnectionGenerator using CGConnect.
```

Comment:
The Description above claims that a "list containing the values" may be pushed to the stack. From the code, it seems that the values are pushed, not a list of them.

Reply:
Fixed in 0066013c9.